doc/source/Container.md

```diff
@@ -22,6 +22,18 @@ $ export CONTAINER_ENV_VARIABLE=CONVERSATION_SERVICE_WATSON_PIZZERIA
 $ export CLUSTER_NAME=<your_cluster_name>
 ```
 
+* Set the Kubernetes environment to work with your cluster:
+
+```
+bx cs cluster-config $CLUSTER_NAME
+```
+
+This output of this command will contain a KUBECONFIG environment variable that must be exported in order to set the context. Copy and paste this output in the terminal window. An example is:
+
+```
+export KUBECONFIG=/home/rak/.bluemix/plugins/container-service/clusters/Kate/kube-config-prod-dal10-<cluster_name>.yml
```

Comment:
Quotes aren't necessary in this example, but including them might prevent someone else from having a headache with spaces, etc.

Reply:
Are you referring to the backticks (which are present) or recommending some quotes (that are not present)?

Reply:
I was referring to quotes around the environment variable's value, to handle spaces in file paths, for example.